drivers/entropy/entropy_litex.c

```diff
@@ -0,0 +1,79 @@
+/*
+ *
+ * Copyright (c) 2019 Antmicro <www.antmicro.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
```

Comment:
The repository-wide styling for this seems to be the following:

```
/*
 * Copyright (c) 2019 Antmicro <www.antmicro.com>
 *
 * SPDX-License-Identifier: Apache-2.0
 */
```

Reply:
Formatting changed.